pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -15,10 +21,8 @@ def data(request):
     return request.param
 
 
-@td.skip_if_no("pyarrow", min_version="0.15.0")
```

Comment:
Note that this one is actually already tested for 0.15.0 as well, while the others only for 0.16. As long as 0.15 is the minimal supported pyarrow version, I think we should keep the individual skips.